upload_artifacts.sh

```diff
@@ -1,21 +1,26 @@
 #!/bin/bash
 
-# Used in .travis.yml
+# This script is used in .travis.yml
+
+set -e
 
 BUCKET=piksi-buildroot-images
 PIKSI_VERSION=v3
+BUILD_VERSION=$(git describe --tags --dirty --always)
 
-set -e
+FOLDER=$TRAVIS_BRANCH
+if [ "$TRAVIS_PULL_REQUEST" == "true" ]; then
+  FOLDER="pull_requests/$FOLDER"
+fi
+
+BUILD_DIR="UTC-$(date -u +%Y-%m-%dT%H:%M:%SZ)_$(echo $TRAVIS_BUILD_NUMBER)_$(echo $BUILD_VERSION)"
```

Comment:
had to do that because otherwise it globs the following `_` onto it and looks for `$TRAVIS_BUILD_NUMBER_`

```
$ export COMMIT=abcdefg
$ export TRAVIS_BUILD_NUMBER=42
$ echo "$TRAVIS_BUILD_NUMBER"; echo "$COMMIT"
42
abcdefg
$ echo "$TRAVIS_BUILD_NUMBER_$COMMIT"
abcdefg

```